.github/pull_request_template.md

```diff
@@ -1,23 +1,34 @@
 # Description
-<!-- Briefly describe the feature or bug that your pull request addresses -->
+<!-- Briefly describe the feature or bug that your pull request addresses, 1-2 sentences. -->
 
-# Check List
 
-- [ ] Are you changing anything with the public API?
-- [ ] Are your changes backwards compatible with previous SDK Versions?
-- [ ] Have you tested this change on real device?
-- [ ] Have you added unit test coverage for your changes?
-- [ ] Have you verified that your changes are compatible with all Android versions the SDK currently supports?
```

Comment:
I noticed recently that the phrasing of these didn't feel quite suited to a checkbox, like is the checkbox me saying yes it contains public API changes? Or just "I've considered this"?

Reply:
Thoughts on an ask for more explicit information around _which_ SDK versions the change was tested?

Reply:
99% of the time it doesn't matter, but yeah maybe this could be phrased to get the author to at least think about whether they need to

Reply:
true!